### ISSUE TITLE
print server REST endpoint only once

### ISSUE THREAD

**Issue by harsham05** [chrismattmann/tika-python#91]
@chrismattmann 

The following is printed to stdout for every document **parse** request to Tika REST server.
```
('server endpoint:', 'http://localhost:9998')
```

https://github.com/chrismattmann/tika-python/blob/master/tika/tika.py#L276

**Comment by chrismattmann** [comment]
can you submit a PR to remove this @harsham05 ?

**Comment by harsham05** [comment]
@chrismattmann 
The print statement is indicative of debugging/sanity reasons. It was **not** present in previous tika-python Releases.  I will submit a PR removing the same.
```
('server endpoint:', 'http://localhost:9998')
```
Tika REST Server runs on port 9998 by default & the same is already mentioned in Readme.

**Comment by chrismattmann** [comment]
fixed in #92

**Comment by harsham05** [comment]
Fixed in https://github.com/chrismattmann/tika-python/pull/92
